.velocitas-lock.json

```diff
@@ -0,0 +1,8 @@
+{
+    "packages": {
+        "devenv-runtimes": "v4.0.5",
+        "devenv-github-workflows": "v6.0.4",
+        "devenv-github-templates": "v1.0.5",
+        "devenv-devcontainer-setup": "v2.4.7"
```

Comment:
I just use the basic stuff, but it might make sense to double check this. Not everything might be useful for us?